Makefile

```diff
@@ -36,19 +33,19 @@ services:
 	docker-compose up -d
 
 .PHONY: dev
-dev: build/manifest.json
+dev: build/manifest.json tox
```

Comment:
`tox` is added as a dependency to every target that requires tox and Python, which means they will be automatically installed when you try to run any of those targets